apps/app/src/routes/_layout/profile/_tabs/my-feeds.tsx

```
@@ -0,0 +1,172 @@
+  }>({ term: "", value: "all" });
+  const navigate = useNavigate();
+
+  const { data: feeds = [], isLoading, error } = useMyCreatedFeeds();
+
+  const filteredFeeds = useMemo(() => {
+    return feeds.filter((feed) => {
+      const matchesSearch =
+        !filters.term ||
+        feed.name.toLowerCase().includes(filters.term.toLowerCase()) ||
+        (feed.description &&
+          feed.description.toLowerCase().includes(filters.term.toLowerCase()));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)